storybook/decorators/with-global-css.js

```diff
@@ -59,7 +54,7 @@ export const WithGlobalCSS = ( Story, context ) => {
 	}, [ context.globals.css ] );
 
 	return (
-		<div className={ clsx( classes ) }>
+		<div className={ classes }>
```

Comment:
This change feels risky in comparison to the others, is this variable guaranteed to be a string?

Reply:
Yeah, I'd likely keep that `clsx` call just in case, since it's not invoked with a static string literal.

Reply:
Good catch :+1: